### PR TITLE
add nbdev flags to test action

### PR DESCRIPTION
**Pull request by filipj8**

### CODE REVIEW

File: nbdev-ci/action.yml [hunk 1]
```diff
@@ -13,6 +13,11 @@ inputs:
     description: 'Skip tests?'
     required: false
     default: ''
+  flags:
+    description: 'Space separated list of nbdev test flags to run that are normally ignored'
+    required: false
+    default: ''
+
```
Comment (filipj8) [inline nbdev-ci/action.yml:20]:
```suggestion
```

**Comment by filipj8** [comment]
Redo of https://github.com/fastai/workflows/pull/56 @jph00

**Comment by jph00** [comment]
Thank you!

**Comment by paulclou** [comment]
From my setup, `default` for `flags` need to be set to `"''"`, in order to send `''` as parameter to `nbdev_test --flags`.

```
  flags:
    description: 'Space separated list of nbdev test flags to run that are normally ignored'
    required: false
    default: "''" <----
```

The current default value (`''`) does not pass anything to `nbdev_test --flags`, leading to the error shown below. I assume this is due to syntax of YAML.

```
nbdev_test: error: argument --flags: expected one argument
Error: Process completed with exit code 2.
```

**Pull request merged**